src/components/DevelopmentGuide/V4MigrationGuide.mdx

```diff
@@ -228,10 +228,27 @@ but it’s possible a page will not look as expected on smaller screen sizes and
   </Box>
 </Flex>
 
-#### Rename `tableTextSize` to `fontSize`.
+#### If using `Table`, rename `tableTextSize` to `fontSize`.
 
 We updated the `tableTextSize` prop in `Table` to `fontSize` to align more closely with the CSS property.
 
+#### If using the `onMixedStateChange` function, adjust the argument you're passing.
+
+We moved to object destructuring for functions with 3+ arguments. This is mostly an internal, non-breaking
+change, but if you're using the `onMixedStateChange` function from the `useMultiSelect` hook, you'll need
+to move the arguments you're passing inside an object.
+
+<Source
+  code={`
+// This no longer works.
+onMixedStateChange(e.target.id, multiSelect.id, multiSelect.items);
+
+// Args must be inside an object.
+onMixedStateChange({e.target.id, multiSelect.id, multiSelect.items});
```

Comment:
1) indent the code block
2) it needs a slight adjument `({ parentId: e.target.id, multiSelectId: multiSelect.id, items: multiSelect.items })`